### PR TITLE
api: add ToggleButton types

### PR DESCRIPTION
**Pull request by landism**
more discussion [here](https://app.shortcut.com/windmill/story/12683/clicking-enable-disable-resource-button-actually-updates-the-resource-s-configmap-disable-value)

### CODE REVIEW

**Comment by landism** [comment]
removing reviewers for now - need to re-create this with api-new-type.sh and clean up comments

**Comment by nicks** [comment]
is this PR waiting on anything? (mainly wanted to make sure it's not blocked on me)

File: pkg/webview/view.swagger.json [hunk 1]
```diff
@@ -156,6 +156,10 @@
         "fieldsV1": {
           "$ref": "#/definitions/v1FieldsV1",
           "title": "FieldsV1 holds the first JSON version format as described in the \"FieldsV1\" type.\n+optional"
+        },
```
Comment (landism) [inline pkg/webview/view.swagger.json:159]: it looks like the changes in this file are because `make proto` wasn't run in #4990

**Comment by landism** [comment]
> is this PR waiting on anything? (mainly wanted to make sure it's not blocked on me)

It was waiting on me to get back around to re-running codegen. I've now added reviewers, so it's waiting on reviews!

**Review by milas — APPROVED** [review]
⏯️

**Pull request merged**